wcivf/apps/people/models.py

```diff
@@ -64,6 +80,15 @@ def get_absolute_url(self):
                 slugify(self.name)
             ])
 
+    def has_biographical_info(self):
+        attrs = ['place_of_birth', 'secondary_school', 'university_undergrad',
+                 'last_or_current_job']
+        attr_count = 0
+        for a in attrs:
+            if getattr(self, a) is not None:
+                attr_count += 1
+        return (attr_count > 1)
```

Comment:
I want more than one attribute - one on its own looks lonely :smile: though it probably could still be more concise!